commerce_coordinator/apps/iap/views.py

```diff
@@ -308,12 +307,10 @@ def get(self, request):
 
     def _get_service(self, configuration):
         """Create a service to interact with google api."""
-        credentials = ServiceAccountCredentials.from_json_keyfile_dict(
-            configuration.get("google_service_account_key_file"),
-            configuration.get("google_publisher_api_scope"),
+        credentials = service_account.Credentials.from_service_account_info(
+            configuration["google_service_account_key_file"],
+            scopes=[configuration["google_publisher_api_scope"]],
```

Comment:
@NawfalAhmed Have we set these keys in edx-internals? If not, let's please do this along with this PR.